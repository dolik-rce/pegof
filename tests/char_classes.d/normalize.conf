input char_classes.d/char_classes.peg
optimize all
