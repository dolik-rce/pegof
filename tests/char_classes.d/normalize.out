main
    <- [0-9A-F]
    / [u-z]
    / [aeiou]
    / [1-9ABa-h]
    / [-\t\n\r&\[-\]]
